app/services/item_service.rb

```diff
@@ -0,0 +1,6 @@
+class ItemService
+  def get_all_items
+    response = Faraday.get("http://localhost:3000/api/v1/items")
```

Comment:
We could probably DRY up some of the Faraday logic, especially the base url `http://localhost:3000/`. Do `ItemService` and `MerchantService` need to be separate? If so, maybe they could inherit a basic connection method from something like `RailsEngineService`.